Quick check/recording data page (#97)
IRL2/nanover-docs: source/concepts/recording.rst

    axis.set_ylabel("Energy (kJ/mol)")

Lower level methods are available in :py:mod:`nanover.mdanalysis.recordings` to read the content of the files directly.
This module is used in the `state-utils <https://github.com/IRL2/nanover-utils>`_ utility that allows to read shared
state recordings in a python script or with the command line.

Recording format
----------------

The current version of the file format is version 2.
Each recording file contains a header and a sequence of records.

The header contains two fields, stored as little endian 8 bytes unsigned integers:

* **a magic number, its value is 6661355757386708963**. This value was chosen arbitrarily and needs to be the first
  8 bytes of the file to indicate it is indeed a NanoVer recording. A file without this magic number is not a NanoVer

>   recording, however one need to keep in mind that a file that starts with that value could still not be a valid

Reviewer: ```suggestion
  recording, however one should keep in mind that a file that starts with that value could still not be a valid
```
